packages/outstatic/src/utils/useFileQuery.ts

```diff
@@ -0,0 +1,30 @@
+import { useContext } from 'react'
+import { OutstaticContext } from '../context'
+import { useDocumentQuery } from '../graphql/generated'
+import { useOstSession } from './auth/hooks'
+
+type useFileQueryProps = {
+  file: string
+  skip?: boolean
+}
+
+const useFileQuery = ({ file, skip = false }: useFileQueryProps) => {
```

Comment:
I really like this hook- there's a lot of places in outstatic where we need a specific file from github.